repose-aggregator/core/repose-core/src/test/java/org/openrepose/nodeservice/request/RequestHeaderServiceContextTest.java

```diff
@@ -55,8 +52,6 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.*;
 
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore("javax.management.*")
```

Comment:
Way back when the test was made in Repose 5.0.0, `PowerMock` was actually being used.
https://github.com/rackerlabs/repose/blob/papi-5.0.0/repose-aggregator/core/core-lib/src/test/java/com/rackspace/papi/service/context/impl/RequestHeaderServiceContextTest.java#L39

It looks like when David was working on Spring, he removed the class that was being Power mocked, so we dropped usage:
https://github.com/rackerlabs/repose/commit/ae1f8b79798427f6427426d3bbd8e0951a5f2fa5#diff-9613b33b12920faafeeb6309e6589818

Since we don't seem to be using PowerMock anymore, we probably don't need those annotations.